Core/Core/Planner/CalendarViewController.swift

```diff
@@ -183,13 +184,19 @@ class CalendarViewController: UIViewController {
         clearPageCache()
         if days.hasDate(date, isExpanded: isExpanded) {
             days.updateSelectedDate(date)
-        } else {
-            let isReverse = date < days.selectedDate
-            // Assumes selected date can't be more than 1 page away
-            let page = daysPageDelta(isReverse ? -1 : 1, from: days)
-            page.loadViewIfNeeded()
-            page.updateSelectedDate(date)
-            daysPageController.setViewControllers([ page ], direction: isReverse ? .reverse : .forward, animated: true)
```

Comment:
this functionality is now missing. This was added to slide over when the list was swiped to a day in a new week/month.

Reply:
Oooohh! That makes sense. I couldn't figure out how to trigger this path. Good to know. Fixed.